containers/unit_tests/TestErrorReporter.hpp

```diff
@@ -50,6 +50,10 @@
 #include <Kokkos_Core.hpp>
 #include <Kokkos_ErrorReporter.hpp>
 
+#ifndef M_PI
+#define M_PI 3.14159265358979323846
+#endif
```

Comment:
Did you try
```C++
#define _USE_MATH_DEFINES
#include <cmath>
```

Reply:
I think I did. The problem is if `<cmath>` is included somewhere else first, `M_PI` is still not defined.